specification/datafactory/resource-manager/Microsoft.DataFactory/stable/2018-06-01/entityTypes/LinkedService.json

```diff
@@ -6710,8 +6710,27 @@
       "properties": {
         "connectionString": {
           "description": "An ODBC connection string. Type: string, SecureString or AzureKeyVaultSecretReference.",
+          "type": "object"
+        },
+        "server": {
           "type": "object",
-          "x-ms-format": "dfe-string"
```

Comment:
Why the dfe-string remvoed from the connectionString. Please reverted it back if by mistake.